kodilla-code-coverage/src/main/java/com/kodilla/hibernate/invoice/Product.java

```diff
@@ -0,0 +1,57 @@
+package com.kodilla.hibernate.invoice;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.util.ArrayList;
+import java.util.List;
+
+@Entity
+@Table(name = "PRODUCTS")
+public class Product {
+    private int id;
+    private String name;
+    private List<Item> items = new ArrayList<>();
+
+    public Product() {
+    }
+
+    public Product(String name) {
+        this.name = name;
+    }
+
+    @Id
+    @GeneratedValue
+    @NotNull
+    @Column(name = "ID", unique = true)
+    public int getId() {
+        return id;
+    }
+
+    @NotNull
+    @Column(name = "NAME")
+    public String getName() {
+        return name;
+    }
+
+    @OneToMany(
+            targetEntity = Item.class,
+            mappedBy = "product",
+            cascade = CascadeType.ALL,
```

Comment:
Cascade all tutaj nie bardzo ma sens. Product jest definicja produktu i przy jej zmianie wszystkie itemy nie musza byc zapisywane
- np Produkt to Jogurt firmy X z danymi parametrami
- Item to 1 sztuka Jogurtu w koszyku (trzyma relacje jaki konkretnie typ Jogurtu, oraz do jakiego rachunku (Invoice'a) jest przypisany)
- Invoice to rachunek z 3 itemami (1 sztuka jogurtu)